graphqlapi/src/main/java/io/stargate/graphql/schema/schemafirst/AdminSchemaBuilder.java

```diff
@@ -0,0 +1,622 @@
+                .type(nonNull(MIGRATION_STRATEGY_ENUM))
+                .defaultValue(MigrationStrategy.ADD_MISSING_TABLES_AND_COLUMNS)
+                .build())
+        .argument(
+            newArgument()
+                .name("force")
+                .description(
+                    "Proceed even if the previous deployment is still marked as in progress. "
+                        + "This is used to recover manually if a previous deployment failed unexpectedly during the "
+                        + "CQL migration phase.")
+                .type(nonNull(GraphQLBoolean))
+                .defaultValue(false)
+                .build())
+        .argument(
+            newArgument()
+                .name("dryRun")
+                .description(
+                    "Just parse and validate the schema, don't deploy it or apply any changes to "
+                        + "the database. This is useful in particular in conjunction with the "
+                        + "`cqlChanges` field in the result, to evaluate the impacts on the CQL "
+                        + "data model.")
+                .type(nonNull(GraphQLBoolean))
+                .defaultValue(false)
+                .build())
+        .type(DEPLOY_SCHEMA_TYPE);
+  }
+
+  private static final GraphQLFieldDefinition DEPLOY_SCHEMA_MUTATION =
+      deploySchemaStart()
+          .name("deploySchema")
+          .description("Deploys a GraphQL schema to a namespace.")
+          .argument(
+              newArgument()
+                  .name("schema")
+                  .description("The contents of the schema.")
+                  .type(nonNull(GraphQLString))
+                  .build())
+          .build();
+
+  private static final GraphQLScalarType UPLOAD_SCALAR =
+      newScalar().name("Upload").coercing(new InputStreamCoercing()).build();
+
+  private static final GraphQLFieldDefinition DEPLOY_SCHEMA_FILE_MUTATION =
+      deploySchemaStart()
+          .name("deploySchemaFile")
+          .description(
+              "Deploys a GraphQL schema to a namespace via a file upload.\n"
+                  + "This mutation must be executed with a [multipart request](https://github.com/jaydenseric/graphql-multipart-request-spec) "
+                  + "(note that your `operations` part **must** declare MIME type `application/json`).")
+          .argument(
+              newArgument()
+                  .name("schemaFile")
+                  .description("The contents of the schema as an UTF-8 encoded file.")
+                  .type(nonNull(UPLOAD_SCALAR))
+                  .build())
+          .build();
+
+  private static final GraphQLObjectType MUTATION =
+      newObject()
+          .name("Mutation")
+          .field(CREATE_NAMESPACE_MUTATION)
+          .field(DROP_NAMESPACE_MUTATION)
+          .field(DEPLOY_SCHEMA_MUTATION)
+          .field(DEPLOY_SCHEMA_FILE_MUTATION)
+          .build();
+
+  private final AuthenticationService authenticationService;
```

Comment:
Does this need to be here? Doesn't seem to be used.

Reply:
Good catch, it's a leftover.